products/rhel7/profiles/rhelh-stig.profile

```diff
@@ -1,11 +1,12 @@
 documentation_complete: true
 
-title: '[DRAFT] DISA STIG for Red Hat Enterprise Linux Virtualization Host (RHELH)'
+title: 'RHV hardening based on STIG for Red Hat Enterprise Linux 7'
 
 description: |-
-    This *draft* profile contains configuration checks that align to the
-    DISA STIG for Red Hat Enterprise Linux Virtualization Host (RHELH).
-    
+    This profile contains configuration checks for
+    Red Hat Enterprise Linux Virtualization Host based on the
```

Comment:
Maybe the following is better?
```suggestion
    Red Hat Virtualization based on the
```

Reply:
That matches the name https://www.redhat.com/en/technologies/virtualization/enterprise-virtualization and https://access.redhat.com/products/red-hat-virtualization so I'm inclined to agree @yuumasato.